test/fixtures/test-runner/output/name_pattern.snapshot

```diff
@@ -171,12 +171,48 @@ ok 15 - no
   duration_ms: *
   type: 'suite'
   ...
-1..15
-# tests 21
-# suites 10
-# pass 13
+# Subtest: DescribeForFullPath
```

Comment:
I didn't find in the [contribution docs](https://github.com/nodejs/node/blob/main/BUILDING.md#running-tests) how to regenerate snapshot file so I updated it manually